install.sh

```diff
@@ -167,6 +167,13 @@ mkdir -p "$OMS_CACHE/runtime-script"
 mkdir -p "$OMS_CACHE/startup-script"
 mkdir -p "$OMS_CACHE/trash"
 
+# install lolcat with apt-get if not found
+if [ ! -f "$OMS/lib/bin/lolcat" ]
+then
+  _info '  Installing lolcat'
+  sudo apt-get install lolcat
```

Comment:
I don't think everyone using `apt`. My computer is Archlinux, and it holds `pacman`.

Reply:
Yeah I understand, I use arch linux too … and kali linux so I have just forgot to add detection of OS and adaptation of the install script.
Let’s me fix that.

Reply:
I think, it can be a notification and just tell users there is an option about lolcat.

Reply:
> I think, it can be a notification and just tell users there is an option about lolcat.

I take note